apps/src/templates/checkbox-dropdown.module.scss

```diff
@@ -65,7 +66,7 @@ form.dropDownMenuContainer {
       gap: 12px;
       margin: 0;
 
-      input[type="checkbox"] {
+      input[type='checkbox'] {
```

Comment:
were these changes just from the linter?

Reply:
Correct, should I change them back?

Reply:
as long as it passes the linter, it's good!